sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -27,6 +29,7 @@
  * Adapter utilities for mapping Raw YAML objects to internal objects.
  */
 public class YAMLToInternalMappers {
+    private static final Logger LOGGER = LoggerFactory.getLogger(YAMLToInternalMappers.class);
```

Comment:
Prefer `non-static` loggers when possible.  Not sure, it's possible here, just a note.